tests/test_jobqueue.py

```diff
@@ -227,7 +227,7 @@ def test_longer_first(self, job_queue):
 
         sleep(0.015)
 
-        assert self.result == 1
+        assert self.result == 0
```

Comment:
This shouldn't be affected by your changes - please revert. If the test fails locally for you, please make sure to synchronize your machines system time. If it still fails, just ignore it - we'll see how the CI performs